### ISSUE TITLE
Icon Request: fa-zillow

### ISSUE THREAD

**Issue by clearly** [FortAwesome/Font-Awesome#2623]
Was thinking of just taking this .eps and converting the House-Z logo part into a font-awesome icon with the proper spacing/sizing. I'm building a site that integrates with Zillow and having this icon in the Font-So-Awesome stable would be really great.

http://zillow.mediaroom.com/file.php/1279/PRESS_zillow_logo_woTag_black.eps

**Comment by zteater** [comment]
+1

**Comment by roadhousemike** [comment]
+1

**Comment by Cutcopy** [comment]
+1

**Comment by jpstearns** [comment]
+1

**Comment by osowebproductions** [comment]
+1

**Comment by jwrussell83** [comment]
+1

**Comment by madhuinc** [comment]
+1

**Comment by katoner** [comment]
+1

**Comment by indybond1** [comment]
+1,000,000,000

**Comment by mlbdenver** [comment]
+1

**Comment by MikeTrottman** [comment]
+1

**Comment by mattie02** [comment]
+1

**Comment by clitnak** [comment]
+1

**Comment by bkubic** [comment]
+1

**Comment by deevoweb** [comment]
Yeah.  Zillow is a public company and a MASSIVE brand.  As is Realtor.com.   Those should definitely both be here.

**Comment by jorgecas99** [comment]
Dear Font Awesome dev team, this request has been opened since 2013 with no track. I don't believe you need to justify adding Zillow. If you live in the USA you know who Zillow is. Huge brand. If the issue is that no companies should be included well that is a different issue, just say so.

**Comment by deevoweb** [comment]
Mega brand and literally thousands and thousands of app/web developers reference them.  Would be nice to have a unified font icon to go along with the other tools real estate agents and developers use.

**Comment by sibyl** [comment]
+1
Most realtors use this. It's as important as any other social media for that industry. Should be added.

**Comment by bobbyacker** [comment]
Need Also

**Comment by johnoverstreet** [comment]
+1
People have been asking for this icon since 2014. Zillow is vital for real estate companies. Gonna find a different font library now.

**Comment by commanderfun** [comment]
+1

**Comment by commanderfun** [comment]
It would be great if FontAwesome had icons for all the top Real Estate companies. Zillow, Trulia, RedFin.

**Comment by jie-qin** [comment]
+1

**Comment by joshgilson** [comment]
+1

**Comment by HeliosMagin** [comment]
+1

**Comment by jeffal** [comment]
+1

**Comment by bokenrosie** [comment]
Add me to the list.  Please offer a fa-zillow logo.

**Comment by sbcr8ive** [comment]
+1 Zillow and Realtor need both!

**Comment by ganya555** [comment]
opened this issue on Dec 20, 2013
6 years.........

**Comment by Erutan409** [comment]
Could a project contributor comment on this?  Seems like this is just being dismissed.

**Comment by khuderm** [comment]
+1

**Comment by melissaportfolio** [comment]
Is there a reason fa-zillow hasn't been implemented in 8 years time?

**Comment by jhned** [comment]
+1 to both Zillow and Realtor.com

**Comment by frasurebuilds** [comment]
+1

**Comment by cherylt327** [comment]
+1   I need the Zillow icon!

**Comment by johnoverstreet** [comment]
I last posted on this thread in 2018 for work, and I've been following it since even after switching positions. There has been no traction from any member of the Font Awesome team, despite commenting on and closing a duplicate issue by pointing to this issue. 

Almost 10 years of radio silence, despite being aware of the need for these icons. Does anyone know if they have posted any information about this somewhere else or something? This is kind of insane to me. 

I'd like to think it is the companies involved not allowing their logos to be used, but at this point even just saying that would be a huge improvement.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
